src/Reflector/ClassReflector.php

```diff
@@ -28,15 +25,13 @@ public function __construct(SourceLocator $sourceLocator)
     }
 
     /**
-     * @return self
+     * @deprecated
```

Comment:
Should we keep this method now that we have a global entry point for the library?

Reply:
I vote for removing.